update.sh

```diff
@@ -97,7 +97,7 @@ function in_variants_to_update() {
   local variant=$1
 
   if [ "${#update_variants[@]}" -eq 0 ]; then
-    echo 0
+    echo 1
```

Comment:
Echo? Not exit?

Reply:
Yeah because it's a function that needs to return a boolean. This causes the script to skip the unrecognized variant